components/details/DetailsCard.jsx

```diff
@@ -0,0 +1,40 @@
+import { useRouter } from "next/router";
+import moment from "moment/moment";
+import ItemCategoryIcon from "../ItemCategoryIcon";
+import Link from "next/link";
+import styled from "styled-components";
+import { DetailsCardStyled, StyledLink, StyledShopLink } from "./DetailsStyled";
+
+export default function DetailsCard({ items }) {
+  function findMe(item) {
+    return item.id === actualRoute;
+  }
+
+  const router = useRouter();
+  const actualRoute = router.query.id;
```

Comment:
renaming the id to actualRoute seems confusing
```suggestion
  const {id} = router.query;
```